README.md

```diff
@@ -166,6 +166,42 @@ feather.icons.x.toSvg({ class: 'foo bar', 'stroke-width': 1, color: 'red' })
 
 See the [API Reference](#api-reference) for more information about the available properties and methods of the `feather` object.
 
+### SVG Sprite
```

Comment:
Awesome, can you add this to the table of contents?

Reply:
yes, sure! I forgot that, I'm not used to contribute to such well made projects.